### PR TITLE
update the admin_gui_auth_conf of the OIDC

### PR DESCRIPTION
**Pull request by raoxiaoyan**
### Description
Waiting for this PR merged.
https://github.com/Kong/kong-ee/pull/6848

 

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_src/gateway/kong-manager/auth/oidc/configure.md
Comment (sumimakito) [inline app/_src/gateway/kong-manager/auth/oidc/configure.md:35]:
```suggestion
While authenticating Kong Manager with OpenID Connect, please ensure that your IdP supports the `authorization_code` grant type and is enabled for the associated client.

Configuration parameters in `admin_gui_auth_conf` for `openid-connect`:

```

### Add a note

To let users know that OpenID Connect with Kong Manager only supports the `authorization_code` grant type.

### Fix the table

Screenshot 2023-10-25 at 14 24 46

The table looks broken. We need to leave a blank line between the paragraph and the table to let the table render normally.

Screenshot 2023-10-25 at 14 28 25

**Review by sumimakito — CHANGES_REQUESTED** [review]

**Review by sumimakito — APPROVED** [review]
LGTM. Please also wait for reviews from members of the Docs team.

File: app/_src/gateway/kong-manager/auth/oidc/configure.md
Comment (cloudjumpercat) [inline app/_src/gateway/kong-manager/auth/oidc/configure.md:21]: @raoxiaoyan Hi! I added some conditional rendering since I think the changes only apply to 3.5.x and greater, right? If that sounds correct, I can approve and merge this :)

Comment (raoxiaoyan) [reply app/_src/gateway/kong-manager/auth/oidc/configure.md:21]: Correct. Thank you.

**Pull request merged**
